documentation/assemblies/configuring/assembly-kafka-entity-operator.adoc

```diff
@@ -1,24 +1,22 @@
 // This assembly is included in the following assemblies:
 //
-// assembly-config-kafka.adoc
+// assembly-config.adoc
 
 [id='assembly-kafka-entity-operator-{context}']
 
-= Configuring the Entity Operator
+= Configuring Strimzi operators
```

Comment:
It is a weird to leave out entity operator here and mention it right in the first sentence. I would probably revert the original name or use something like `Configuring Strimzi User and Topic operators`.

Reply:
Reverted